test/mason/mason-external/libtomlc99/mason-external.skipif

```diff
@@ -0,0 +1 @@
+../../suseSkipif
```

Comment:
Note: I did not hardcode the compiler into this version, so this test could potentially run on all configurations. I am leaving this restriction in for now though, to reduce the chances of testing noise in case it isn't.